tests/fuzz/fuzz_h2_decoder_correct.c

```diff
@@ -29,12 +29,45 @@
 static const uint32_t FRAME_PREFIX_SIZE = 3 + 1 + 1 + 4;
 static const uint32_t MAX_PAYLOAD_SIZE = 16384;
 
-static struct aws_http_headers *s_generate_headers(struct aws_allocator *allocator, struct aws_byte_cursor *input) {
+enum header_style {
+    HEADER_STYLE_REQUEST,
+    HEADER_STYLE_RESPONSE,
+    HEADER_STYLE_TRAILER,
+};
+
+static struct aws_http_headers *s_generate_headers(
+    struct aws_allocator *allocator,
+    struct aws_byte_cursor *input,
+    enum header_style header_style) {
 
     struct aws_http_headers *headers = aws_http_headers_new(allocator);
 
-    /* Requires 4 bytes: type, size, and then 1 each for name & value */
-    while (input->len >= 4) {
+    /* There are pretty strict requirements about pseudo-headers, no randomness for now */
+    if (header_style == HEADER_STYLE_REQUEST) {
+        struct aws_http_header method = {.name = aws_http_header_method, .value = aws_http_method_get};
```

Comment:
Would it be reasonable to swap between a list of verbs for method, maybe toggle between http/https for scheme, and generate domains from usable characters?

Reply:
yeah it's reasonable, I was just trying to get this working again as fast as possible.
I'm going to keep it as-is. This doesn't feel like a high value change right now